webpack/scenes/ContentViews/ContentViewsConstants.js

```diff
@@ -56,9 +56,9 @@ export const addComponentSuccessMessage = component => (component ? __('Updated
 
 
 // Repo added to content view status display and key
-export const ADDED = 'Added';
-export const NOT_ADDED = 'Not added';
-export const ALL_STATUSES = 'All';
+export const ADDED = __('Added');
```

Comment:
these statuses are used as responses to the API in a few places. 
For example in ContentViewComponents.js the "statusSelected" value is one of these translated values. 
We send this directly to the api within the getContentViewComponents functions "status" param. 
Unless the API expects translated values we need to change all usages of these variables to constants. 
An enum pattern like 
`
const STATUSES = { 
added: __('Added'),
not_added: __('Not Added'),
...
}
`
The keys will obviously have to reflect whatever the api expects for statuses of course.